docs/FAQ/FAQ_How_do_I_specify_the_default_type_of_native_renderer_that_is_used_by_the_Browser.md

```diff
@@ -7,7 +7,7 @@ A user can set a property to specify the type of native renderer to use for `SWT
 
 _Note: As of Eclipse/SWT 4.8, Mozilla (`XULRunner`) renderer is no longer supported, the value `mozilla` has no effect._
 
-A user can specify a comma-separated list of native renderers, in order of preference, for the `org.eclipse.swt.browser.DefaultType` value. Values not applicable to a particular platform are ignored. For example, the value of `edge,chromium` will change the default to Edge on Windows and Chromium on other platforms.
+A user can specify a comma-separated list of native renderers, in order of preference, for the `org.eclipse.swt.browser.DefaultType` value. Values not applicable to a particular platform are ignored. For example, the value of `ie,chromium` will change the default to Internet Explorer on Windows and Chromium on other platforms.
```

Comment:
seeing as chromium is not supported anymore, should we replace this constant with `webkit`?

Reply:
Please do so.

Reply:
Done. I also added the deprecation information to the where the CHROMIUM flag is mentioned.